tests/jballerina-semtype-test/src/test/java/io/ballerina/test/SemTypeTest.java

```diff
@@ -81,7 +82,7 @@ public void testSubTypeRelationship(String filePath) throws IOException {
         List<BTypeDefinitionSymbol> bTypeDefinitionSymbols = new ArrayList<>();
         for (Scope.ScopeEntry value : bLangPackage.symbol.scope.entries.values()) {
             BSymbol bSymbol = value.symbol;
-            if (bSymbol.kind == SymbolKind.TYPE_DEF) {
+            if (bSymbol.kind == SymbolKind.TYPE_DEF && bSymbol.origin == SymbolOrigin.SOURCE) {
```

Comment:
Shouldn't this be ! Virtual?

Reply:
We are going to test the relations among the type definitions defined in the source file.  So the origin should be `source` right? (It cannot be `BUILTIN`)

Reply:
If it doesn't consider balas; it's fine.